test/reusing-processes-2/README

```diff
@@ -0,0 +1 @@
+Check if processes are being reused
```

Comment:
It would be better if README files were more precise, at least not copies of each other.

Reply:
Fixed